### ISSUE TITLE
[GitHub::Accessibility::NoTitleAttribute] False positive on <link> tag

### ISSUE THREAD

**Issue by christinach** [github/erblint-github#108]
We (@sandbergja and @christinach) would expect this rule not to raise an error for the `<link>` tag. 
The specific link element is:
`<link rel="unapi-server" type="application/xml" title="unAPI" href="/unapi"/>`

Here is the error message we received:
```
GitHub::Accessibility::NoTitleAttribute:The title attribute should never be used unless for an `<iframe>` as it is inaccessible for several groups of users.
Learn more at https://github.com/github/erblint-github#rules.

<link rel="unapi-server" type="application/xml" title="unAPI" href="/unapi"/> (GitHub::Accessibility::NoTitleAttribute)
```
See reference in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#title)
[A link tag does not display to the user](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). It's more for linking external resources.
